RNTester/js/RNTesterExampleFilter.js

```diff
@@ -77,6 +78,11 @@ class RNTesterExampleFilter extends React.Component<Props, State> {
             this.setState(() => ({filter: text}));
           }}
           placeholder="Search..."
+          placeholderTextColor={
```

Comment:
Splitting the text and background color changes in to a different branch would certainly be a cleaner fix. Only issue with putting a bunch of discrete changes into one PR is the history of changes can get confusing (as this, by title, is a scroll indicator fix).

Reply:
I agree, but its a minor cosmetic change to the test app.   If I didnt have it, the animated gifs would be ugly :)